src/Common/src/TypeSystem/Interop/MethodDesc.Interop.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.Runtime.CompilerServices;
+using System.Reflection.Metadata;
```

Comment:
We need a better abstraction here.

As a general guideline, `using System.Reflection.Metadata` is only allowed in files that have `Ecma` in the file name - we'll not be able to implement `GetPInvokeMethodParameters` on anything but `EcmaMethod` if it stays like this, but we'll want to support it on e.g. `NativeFormatMethod` at some point.